ledger/eval/eval.go

```diff
@@ -1163,6 +1188,7 @@ func (eval *BlockEvaluator) transaction(txn transactions.SignedTxn, evalParams *
 		}
 
 		// Transaction already in the ledger?
+		// this checks against the txns added to this evaluator; testTransaction currently only checks against committed txns.
```

Comment:
> // this checks against the txns added to this evaluator; testTransaction currently only checks against committed txns.

This is not 100% accurate, `checkDup` checks the ledger as well, maybe reword to "against the txns added to this evaluator and the ledger" ?